core/modules/system/css/messages.theme.css

```diff
@@ -100,33 +106,11 @@ div.messages.error:before {
   color: #004570;
 }
 div.messages.info:before {
-  background-image: url(../../../misc/message-24-info.png);
+  background-image: url(../../../misc/message-32-info.png);
   background-color: #0074bd;
 }
 
-.messages a {
-  text-decoration: underline;
-}
-.messages a:hover {
-  text-decoration: none;
-}
-
-/* Desktop size icons */
-@media only screen and (min-width: 34em) {
-  div.messages.status:before {
-    background-image: url(../../../misc/message-32-ok.png);
-  }
-  div.messages.warning:before {
-    background-image: url(../../../misc/message-32-warning.png);
-  }
-  div.messages.error:before {
-    background-image: url(../../../misc/message-32-error.png);
-  }
-  div.messages.info:before {
-   background-image: url(../../../misc/message-32-info.png);
-  }
-}
-/* SVG overrides */
+/* SVG overrides
```

Comment:
Looks like you disabled the SVGs entirely here.

Reply:
oops, that was accidental. for testing. Fixing now!

Reply:
Fixed :)